src/System.Threading/tests/MutexTests.cs

```diff
@@ -70,11 +70,8 @@ public void Ctor_NameUsedByOtherSynchronizationPrimitive_Windows()
             }
         }
 
-        [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(
-            TargetFrameworkMonikers.Uap,
-            "Impersonation APIs are not available and creating global sync objects is not allowed in UWP apps.")]
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWinRT))] // Can't create global objects in appcontainer
```

Comment:
Exactly and I'm pretty sure that's why@danmosemsft used it instead of checking the target framework. It makes more sense and IMO it should always be preferred when hitting AppContainer limitations.